tools/container/huelb/hue_httpd_template

```diff
@@ -17,6 +18,16 @@ LogLevel warn
   CustomLog "${HUE_LOG_DIR}/httpd_access_log" common
 </IfModule>
 
+<IfModule mpm_event_module>
+  ServerLimit             5000
+  StartServers            1000
```

Comment:
Does this also need any updates on the `resources` requirements for the huelb pods?

Reply:
I ran the tests on rhel8 3node cluster. The CPU utilization was at 70-80% for 1000 concurrent threads (huelb-> hue_server). When hitting the hue_server directly I observed similar utilization. So I think we are good with the current resource requirements.